buildscripts/provisioner-nfs/Dockerfile

```diff
@@ -12,7 +12,7 @@
 #See the License for the specific language governing permissions and
 #limitations under the License.
 #
-FROM alpine:3.11.5
+FROM alpine:3.12
```

Comment:
Info: Changes in this file are done to sync with [buildx](https://github.com/openebs/dynamic-nfs-provisioner/blob/347ab2f6c10d8faeb209743bd0dfe6d65d76a52f/buildscripts/provisioner-nfs/provisioner-nfs.Dockerfile#L43) docker file.

Reply:
:heavy_check_mark: